docs/supported_apis/dataframe_supported.rst

```diff
@@ -1,676 +1,45 @@
-+----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``tz_convert``             | `tz_convert`_             | Y                      |                                                    |
-+----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``tz_localize``            | `tz_localize`_            | Y                      |                                                    |
-+----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``unstack``                | `unstack`_                | Y                      |                                                    |
-+----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``update``                 | `update`_                 | Y                      |                                                    |
-+----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``values``                 | `values`_                 | Y                      |                                                    |
-+----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``value_counts``           | `value_counts`_           | D                      |                                                    |
-+----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``var``                    | `var`_                    | P                      | Modin defaults to pandas if given the ``level``    |
-|                            |                           |                        | param.                                             |
-+----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``where``                  | `where`_                  | Y                      |                                                    |
-+----------------------------+---------------------------+------------------------+----------------------------------------------------+
+.. table::
+   :widths: 1, 5
+
+   +-------------+-----------------------------------------------------------------------------------------------+
+   | Flag        | Meaning                                                                                       |
+   +=============+===============================================================================================+
+   | Supported   | Parameter is supported, it's usage brings performance improvement                             |
```

Comment:
Is this flag applicable to methods, isn't?

Reply:
The idea was to use this flag in the Supported APIs table in the cases when method is fully supported, but it will deviate from suported parameters tables rule in the docstrings (where we leave supported parameter cell empty), so this flag was removed.